en/reference/annotations-mapping.rst

```diff
@@ -50,7 +50,7 @@ Optional attributes:
 
 -  **nodeType**: PHPCR type for this node, default ``nt:unstructured``.
 -  **repositoryClass**: Name of the repository to use for this document.
--  **versionable**: Set to true to enable versioning, implies *referenceable*.
+-  **versionable**: Set to "simple" or "full" to enable versioning (respectively simple and full level), implies *referenceable*.
```

Comment:
it can also be ``false``: https://github.com/doctrine/phpcr-odm/blob/master/doctrine-phpcr-odm-mapping.xsd -- @dbu @lsmith77 is that to disable the inheritance of versioning?